### ISSUE TITLE
Segmentation result display

### ISSUE THREAD

**Issue by zedyasuotalong** [VendenIX/BrainMetaSegmentatorUI-Front#3]
Hi,
I tried to integrate my own two-dimensional segmentation model and found that the console logs were able to correctly segment, but the front page still couldn't display any segmentation results.
Therefore, I think something may be wrong with my integrate way.
So, I assigned a segmentation result directly on your initial code, which means the input CT data doesn't go through the model, but the front page still can't display the result. I would like to know why this is happening and if there are any steps that haven't been set up properly. 
Here are some debug log for reference:
1.result assignment:
<img width="861" alt="8fd3cbf5dce66e1d05698732f4f3dfe" src="https://github.com/user-attachments/assets/066cdc03-3e45-4af4-bc34-66853fefcd3c">
2.the back-end log:
<img width="674" alt="043984991645ec91b7146304dd2b08e" src="https://github.com/user-attachments/assets/72b80d3b-5233-48f1-a4e4-aae98ea47c3b">
3.the front-end page:
![image](https://github.com/user-attachments/assets/b5e831b9-5ac8-45d6-b118-d5f74c93f5d9)

**Comment by VendenIX** [comment]
Hi, your mock code work for me (tested on the branch deploiementWindows)
![Screenshot 2024-08-06 111036](https://github.com/user-attachments/assets/5a333c1c-fef7-451a-802a-a42b96f5932f)

This is strange, your back-end logs are good.

**Comment by zedyasuotalong** [comment]
Hi,
After "yarn install", I start the front-end in the following way: yarn run dev:orthanc
Is there a problem with starting in this way? Because I found that this way is different from the startup method "yarn start" in your README

Looking forward to your reply, thank you

**Comment by VendenIX** [comment]
Nah, the problem isn't with the front end; it's probably your data. I have the same issue with some data, and I don't yet know what causes it. It could be a history of DICOM tags. (Personally i use yarn start for the developpement and it works)
